transit_odp/timetables/transformers.py

```diff
@@ -69,9 +77,22 @@ def transform(self) -> TransformedData:
             df_merged_vehicle_journeys = merge_vehicle_journeys_with_jp(
                 vehicle_journeys, journey_patterns
             )
-            journey_patterns = merge_journey_pattern_with_vj_for_departure_time(
-                vehicle_journeys.reset_index(), journey_patterns
-            )
+            if is_timetable_visualiser_active:
+                df_merged_vehicle_journeys_with_lines = (
```

Comment:
If we reuse `vehicle_journeys` instead of `df_merged_vehicle_journeys_with_lines` we can remove the else condition in the if block and move merge_journey_pattern_with_vj_for_departure_time outside the if block